controlplane/pkg/prefix_pool/prefixpool.go

```diff
@@ -34,6 +35,8 @@ type prefixPool struct {
 }
 
 func (impl *prefixPool) GetPrefixes() []string {
+	/* Sort the prefixes, so their order is consistent during unit testing */
```

Comment:
Would it be better to do on split to store sorted list of prefixes? As I remember original algorithm was created a list of sorted prefixes by design?

Reply:
Thanks, I've moved the sorting to be applied during the release phase 👍